TAO/orbsvcs/orbsvcs/Naming/Naming_Server.h

```diff
@@ -131,19 +136,31 @@ class TAO_Naming_Serv_Export TAO_Naming_Server
   /// Returns the IOR of the naming service.
   char * naming_service_ior (void);
 
-  /// Returns a NamingContext_ptr for the root Naming Context.
+  /// Returns a <NamingContext_ptr> for the root Naming Context.
   CosNaming::NamingContext_ptr operator-> (void) const;
 
 protected:
-  /**
-   * Helper method: create Naming Service locally. Can be specialized to
+  struct IOR_Bundle {
+    CORBA::Object_var ref_;
+    ACE_CString ior_;
+    ACE_TString filename_;
+  };
+
+  void assign (size_t ndx, bool take, CORBA::Object_ptr obj);
+  int write (size_t ndx) const;
+
+  enum Base_IOR_Indexes {
+    ROOT = 0
+  };
+
+  /**   * Helper method: create Naming Service locally. Can be specialized to
    * refine how Naming Service components are created and initialized
    * Make the root context of size
-   * @a context_size, register it under the @a root_poa, and make the Naming
-   * Service persistent if @a persistence_location is not 0.
-   * (@a persistence_location specifies name of the file to use for
+   * <context_size>, register it under the <root_poa>, and make the Naming
```

Comment:
Using <> in the comments causes doxygen warnings, @a is the way to refer to arguments with doxygen